chaos-monkey-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -1,15 +1,15 @@
 [[getting-started]]
 == Getting started ==
-How to activate Chaos Monkey for Spring Boot in your existing Spring Boot App?
+There are two ways to activate Chaos Monkey for Spring Boot (CM4SB) into your existing Spring Boot app. Either it is an integral part of your project and is added to your regular dependencies (eg. `pom.xml/build.gradle(.kts)`) or you can include it as an external dependency when starting your Spring Boot App.
```

Comment:
Wording "activate into" could be improved

Reply:
I'll agree, tried to improve it by simply changing "into" to "in"